src/Servers/Kestrel/Core/src/KestrelServerOptions.cs

```diff
@@ -23,11 +23,15 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Core
     /// </summary>
     public class KestrelServerOptions
     {
+        private const string DisableAutoChunkingKey = "Switch.Microsoft.AspNetCore.Server.Kestrel.DisableAutoChunking";
```

Comment:
Can we pick a different name for this? This incorrectly implies Kestrel doesn't do auto-chunking by default.

"Switch.Microsoft.AspNetCore.Server.Kestrel.DisableManualChunking" would be a better name, since what this is really doing is taking away the ability to manually write the chunk framing when you effectively opt out of Kestrel's auto chunking by setting the Transfer-Encoing header.

Reply:
The polarity of DisableManualChunking is backwards, the default for a switch needs to be false.

Reply:
EnableManualChunking?

Reply:
> EnableManualChunking?

Sounds good to me.